src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/EServicesApiServiceImpl.scala

```diff
@@ -83,14 +83,16 @@ final case class EServicesApiServiceImpl(
     val result = for {
       requesterId       <- getOrganizationIdFutureUUID(contexts)
       apiStates         <- parseArrayParameters(states).traverse(EServiceDescriptorState.fromValue).toFuture
+      producersUuids    <- parseArrayParameters(producersIds).traverse(_.toFutureUUID)
```

Comment:
IMO this doesn't need `Future.traverse` but I can add it if there are different opinions

Reply:
Sounds good to me.